examples/tabs/palette-exemplar/src/ducks/index.ts

```diff
@@ -46,9 +47,10 @@ export interface IState {
  * Dispatcher
  */
 export interface IDispatchProps
-  extends IDispatchSimpleForm,
-    IDispatchSimpleNetwork,
-    IDispatchPaletteExemplar {}
+  extends IRouterProvidedProps,
```

Comment:
The needed change for any Misk-Web tab that now wants typed access to `history, location, or match`

Reply:
nit: can we alphabetize these